tests/unit/test_download.py

```diff
@@ -175,38 +177,33 @@ def test_get_file_io_write_task(self):
 class TestDownloadSpecialFilenameOutputManager(BaseDownloadOutputManagerTest):
     def setUp(self):
         super(TestDownloadSpecialFilenameOutputManager, self).setUp()
-        self.osutil = AlwaysInicatesSpecialFileOSUtils()
+        self.osutil = AlwaysIndicatesSpecialFileOSUtils()
         self.download_output_manager = DownloadSpecialFilenameOutputManager(
             self.osutil, self.transfer_coordinator,
             io_executor=self.io_executor)
 
     def test_is_compatible_for_special_file(self):
         self.assertTrue(
             self.download_output_manager.is_compatible(
-                self.filename, AlwaysInicatesSpecialFileOSUtils())
-        )
+                self.filename, AlwaysIndicatesSpecialFileOSUtils()))
 
     def test_is_not_compatible_for_non_special_file(self):
         self.assertFalse(
             self.download_output_manager.is_compatible(
-                self.filename, OSUtils())
-        )
+                self.filename, OSUtils()))
 
     def test_get_fileobj_for_io_writes(self):
         with self.download_output_manager.get_fileobj_for_io_writes(
                 self.future) as f:
             # Ensure it is a file like object returned
             self.assertTrue(hasattr(f, 'read'))
-            self.assertTrue(hasattr(f, 'seek'))
             # Make sure the name of the file returned is the same as the
             # final filename as we should not be writing to a temporary file.
             self.assertEqual(f.name, self.filename)
 
     def test_get_final_io_task(self):
```

Comment:
Might be good to add some of the tests from the non-seekable test cases to make sure it behaves as what we would expect as well, specifically the io task submission happens in order of offset.